lib/seahorn/Smt/MarshalYices.cc

```diff
@@ -634,10 +634,8 @@ Expr marshal_yices::eval(Expr expr, ExprFactory &efac, ycache_t &cache,
   Expr domain = nullptr;
   Expr range = nullptr;
 
-  if (bind::isBoolConst(expr) || bind::isIntConst(expr) ||
-      bind::isRealConst(expr) || op::bv::isBvConst(expr)) {
-    // do nothing
-  } else if (bind::isConst<ARRAY_TY>(expr)) {
+  /* special handling for const array*/
```

Comment:
technically, this is just array, not const-array. That is, constant of type array, as opposed to an array that has one constant value for all entries